site/content/docs/5.3/forms/layout.md

```diff
@@ -15,7 +15,7 @@ Every group of form fields should reside in a `<form>` element. Bootstrap provid
 
 Since Bootstrap applies `display: block` and `width: 100%` to almost all our form controls, forms will by default stack vertically. Additional classes can be used to vary this layout on a per-form basis.
 
-## Utilities
+## Sass utilities API
```

Comment:
Since this doesn't refer to the API itself, and rather to classes generated by the API, I'd suggest leaving this as `Utilities`.

```suggestion
## Utilities
```

Reply:
Fwiw the rest of them I'm fine leaving as you've changed them, just this one felt off to me.